packages/@react-stately/slider/src/useSliderState.ts

```diff
@@ -63,7 +63,7 @@ export const DEFAULT_MIN_VALUE = 0;
 export const DEFAULT_MAX_VALUE = 100;
 export const DEFAULT_STEP_VALUE = 1;
 
-export function useSliderState(props: SliderProps): SliderState {
+export function useSliderState(props: Omit<SliderProps, 'direction'>): SliderState {
```

Comment:
Should we keep direction in the Spectrum specific props rather than in the base type?

Reply:
But this is LTR/RTL (and not horizontal/vertical) and it's handled in the aria hooks?

Reply:
Hmm I don't think `direction` should be a prop in the first place though. The aria hooks have access to it via `useLocale`, and since it's being omitted here, I guess stately doesn't need it.

Reply:
Ah, you're right. I was under the impression that useLocale is `@react-spectrum`....